scripts/CEE/worker-node-operations/script.sh

```diff
@@ -123,8 +123,8 @@ reboot_worker(){
 
     drain_worker "${FORCEDRAINMODE}"
 
-    echo "Rebooting node ${WORKER}..."
-    cat <<EOF | oc -n default debug node/$WORKER
+    echo "Rebooting node \"${WORKER}\"..."
+    cat <<EOF | oc -n default debug node/"$WORKER"
```

Comment:
Will this work as expected? I am not sure if we add quote when do oc debug node in general.
It can be 
```
oc -n default debug node/${WORKER}
```
?

Reply:
yes it is working as expected